manager/test_containers/baseimage/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM llparse/vm-base:930a055
```

Comment:
I thought we're going to based on busybox?

Reply:
Yeah, that is literally just a pointer to busybox. I'll change it